modules/core/src/lib/layer-manager.js

```diff
@@ -208,7 +209,7 @@ export default class LayerManager {
   // Update layers from last cycle if `setNeedsUpdate()` has been called
   updateLayers(animationProps = {}) {
     if ('time' in animationProps) {
-      this.context.time = animationProps.time;
+      this.context.timeline.setTime(animationProps.time);
```

Comment:
I presume I am wrong, but I was under the impression that timelines could be added directly to animation loop and have time synced automatically?